kolibri/utils/version.py

```diff
@@ -186,6 +191,31 @@ def get_git_changeset():
             )
 
 
+def git_tagged_version():
+    """
+    :returns: None if no git tag available (no git, no tags, or not in a repo)
+              True if we are exactly on a git tag with the current commit
+              False if we are not.
+    """
+    repo_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    try:
+        p = subprocess.Popen(
+            "git describe --tags --exact-match",
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            shell=True,
+            cwd=repo_dir,
+            universal_newlines=True
+        )
```

Comment:
Does this command fix something that didn't work in `get_git_describe` ? I was just looking at the output of `git describe --tags --match 'v[0-9]*'` and it seems that it also detects the tag for `v0.7.0-beta4`.